rotkehlchen/history/events/structures/asset_movement.py

```diff
@@ -172,9 +137,29 @@ def deserialize_from_db(cls: type['AssetMovement'], entry: tuple) -> 'AssetMovem
             asset=Asset(entry[6]).check_existence(),
             amount=amount,
             extra_data=cls.deserialize_extra_data(entry=entry, extra_data=entry[11]),
-            given_notes=entry[8],
+            notes=entry[8],
         )
 
+    def serialize(self) -> dict[str, Any]:
+        """Serialize the event for api.
+        Autogenerates the event notes, appending any notes added by the user in a second sentence.
+        """
+        serialized_data = super().serialize()
+        location_name = get_formatted_location_name(self.location)
+        asset_symbol = self.asset.resolve_to_asset_with_symbol().symbol
```

Comment:
Maybe the asset has only name and not symbol?

```suggestion
        asset_symbol = self.asset.symbol_or_name()
```

But both of them can raise `UnknownAsset`. In theory. Can this too? Or since it's an already initialized AssetMovement it should not. If so then specify in a comment to make sure it's understood by the reader.

Reply:
Added a note to the docstring mentioning that it can raise UnknownAsset if for some reason the asset is changed after the event is initialized (bypassing the `check_existence` in the init function)